kinetic/distribution.yaml

```diff
@@ -763,6 +763,22 @@ repositories:
       url: https://github.com/mdrwiega/depth_nav_tools.git
       version: kinetic-devel
     status: developed
+  depthcloud_encoder:
+    doc:
+      type: git
+      url: https://github.com/RobotWebTools/depthcloud_encoder.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/RobotWebTools-release/depthcloud_encoder-release.git
+      version: 0.0.5-0
+    source:
+      test_pull_requests: true
```

Comment:
I am not sure you have followed [these instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) for the `RobotWebTools` org unit yet?